drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb

```diff
@@ -141,8 +163,17 @@ def build_assessments(enrollment_scope:, data_collection_stages:, unique_by_info
 
         # Build FormProcessor with IDs to all related records
         assessment.build_form_processor(**value)
-        assessment.save!
+
+        if assessment.exit? && value[:exit_id].nil?
+          # There appear to be lots of "exit" data-collection-stage records for enrollments that don't have an Exit record. Skip those
+          Rails.logger.info "Skipping Exit Assessment for open enrollment. EnrollmentID: #{assessment.enrollment_id}"
+          skipped_exit_assessments += 1
```

Comment:
@eanders I believe this is likely an issue where a client was exited and then un-exited. The data associated with the Exit Data Collection Stage remains, but the Exit record does not. Or, maybe there's something else going on around Exit records.

In this case –– like in the case where there are duplicate records for a given assessments –– I think we should delete those records. I'm holding off on that right now until I get confirmation.